src/Open3D/GUI/Materials/normals.mat

```diff
@@ -15,8 +15,11 @@ vertex {
     void materialVertex(inout MaterialVertexInputs material) {
         gl_PointSize = 3.0;
 
-        float4 normalModel = material.color;
-        material.normalWorld = getViewFromWorldMatrix()*getWorldFromModelMatrix()*normalModel;
+        float4 tangentSpace = getCustom0();
```

Comment:
I think a comment here might be helpful on how you'd like to use the tangent directly to avoid duplicating the mesh, but Filament doesn't let you but fortunately you can get around it using custom0. I'm imagining someone reading this and saying "why not just use the normals" like I did earlier :)